monai/apps/auto3dseg/auto_runner.py

```diff
@@ -278,6 +278,9 @@ def __init__(
         self.set_ensemble_method(self.ensemble_method_name)
         self.set_num_fold(num_fold=self.num_fold)
 
+        self.gpu_customization = False
+        self.gpu_customization_specs = {}
```

Comment:
Flake 8 failure indicates you need something like:
```python
self.gpu_customization_specs: Dict[str, Any] = {}
```
And please run the following line in bash check flake8 locally if you'd like to.
```
./runtests.sh --flake8
```

Reply:
thanks, this is addressed in c01bd76e2550734bc2df5405b02016e9f0ddd652